docs/source/production_deployment/202-bare-metal.rst

```diff
@@ -0,0 +1,20 @@
+Bare Metal Fabric Deployment
```

Comment:
The content below is not strictly about "Bare Metal Fabric Deployment". In fact there is more kubernetes considerations below than bare metal considerations!